### PR TITLE
KOGITO-606 - merged graphql and rest server on same port

### PR DESCRIPTION
**Pull request by Sara4994**

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@Sara4994 perhaps just update the commit message for KOGITO-556?

File: packages/management-console/server/MockData/types.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        processName: String
+        parentProcessInstanceId: String
+        rootProcessInstanceId: String
+        rootProcessId: String
+        roles: [String!]
+        state: ProcessInstanceState!
+        managementEnabled: Boolean
+        endpoint: String!
+        nodes: [NodeInstance!]!
+        error: String
```
Comment (cristianonicolai) [inline packages/management-console/server/MockData/types.js]: error will be a complex type: 

`type ProcessIntanceError {
   nodeDefinitionId: String!
   message: String
}`

```suggestion
        error: String
```

File: packages/management-console/src/components/Molecules/DataListItemComponent/DataListItemComponent.tsx [hunk 2]
```diff
@@ -207,7 +204,7 @@ const DataListItemComponent: React.FC<IOwnProps> = ({ id, instanceID, instanceSt
             id="kie-datalist-action"
             aria-label="Actions"
           >
-            {state === "ERROR" && managementEnabled ? 
+            {state === "ERROR"  && managementEnabled ?
```
Comment (cristianonicolai) [inline packages/management-console/src/components/Molecules/DataListItemComponent/DataListItemComponent.tsx]: this logic needs to be updated. managementEnabled means trigger and retry buttons should be enabled. Error message popup will always be visible.

**Review by cristianonicolai — CHANGES_REQUESTED** [review]

File: packages/management-console/server/MockData/types.js (hunk 1 above)
Comment (Sara4994) [reply packages/management-console/server/MockData/types.js]: modified

File: packages/management-console/src/components/Molecules/DataListItemComponent/DataListItemComponent.tsx (hunk 2 above)
Comment (Sara4994) [reply packages/management-console/src/components/Molecules/DataListItemComponent/DataListItemComponent.tsx]: @cristianonicolai  this is updated

**Review by cristianonicolai — APPROVED** [review]

**Review by AjayJagan — APPROVED** [review]

**Pull request merged**